README.md

```diff
@@ -15,6 +15,27 @@ This pipeline is a re-implementation of the [SARS_Cov2_consensus-nf](https://git
 
 The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible.
 
+## Pipeline summary
+
+1. Raw read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
+2. Adapter trimming ([`Trimmomatic`](http://www.usadellab.org/cms/?page=trimmomatic))
+3. Removal of host reads ([`Kraken2`](http://ccb.jhu.edu/software/kraken2/))
+4. Amplicon removal ([`iVar`](https://github.com/andersen-lab/ivar); *amplicon data only*)
```

Comment:
It's primer removal, not amplicon removal, we remove the primers included in the reads from the amplicons.